http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -605,35 +613,38 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
 
   def internalStartCluster(userEmail: WorkbenchEmail, cluster: Cluster): Future[Unit] = {
     if (cluster.status.isStartable) {
-      val welderDeploy = shouldDeployWelder(cluster)
-      val welderUpdate = shouldUpdateWelder(cluster)
-      for {
-        // Check if welder should be deployed
-        updatedCluster <- if (welderDeploy || welderUpdate) updateWelder(cluster).unsafeToFuture() else Future.successful(cluster)
-
-        // Add back the preemptible instances
-        _ <- if (updatedCluster.machineConfig.numberOfPreemptibleWorkers.exists(_ > 0))
-               gdDAO.resizeCluster(updatedCluster.googleProject, updatedCluster.clusterName, numPreemptibles = updatedCluster.machineConfig.numberOfPreemptibleWorkers)
-             else Future.unit
-
-        // Start each instance individually
-        _ <- Future.traverse(updatedCluster.nonPreemptibleInstances) { instance =>
-          // Install a startup script on the master node so Jupyter starts back up again once the instance is restarted
-          instance.dataprocRole match {
-            case Some(Master) =>
-              googleComputeDAO.addInstanceMetadata(instance.key, getMasterInstanceStartupScript(updatedCluster, welderDeploy, welderUpdate)) >>
+      if (isClusterBeforeCutoffDate(cluster)) {
+        Future.failed(ClusterOutOfDateException())
```

Comment:
I just realized this would block old clusters to be started in _firecloud_, which is not what we want. Probably needs to change to only do this if we're trying to deploy welder.